[2021-02-06 06:42:45,542] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_2 2021-02-06T06:42:42.660009+00:00 [queued]>
[2021-02-06 06:42:45,556] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_2 2021-02-06T06:42:42.660009+00:00 [queued]>
[2021-02-06 06:42:45,556] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-02-06 06:42:45,556] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-02-06 06:42:45,556] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-02-06 06:42:45,565] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): runme_2> on 2021-02-06T06:42:42.660009+00:00
[2021-02-06 06:42:45,569] {standard_task_runner.py:52} INFO - Started process 66 to run task
[2021-02-06 06:42:45,572] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'runme_2', '2021-02-06T06:42:42.660009+00:00', '--job-id', '6', '--pool', 'default_pool', '--raw', '--subdir', '/home/airflow/.local/lib/python3.8/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpw4_8esot', '--error-file', '/tmp/tmpwqg7r5ko']
[2021-02-06 06:42:45,572] {standard_task_runner.py:77} INFO - Job 6: Subtask runme_2
[2021-02-06 06:42:45,608] {logging_mixin.py:104} INFO - Running <TaskInstance: example_bash_operator.runme_2 2021-02-06T06:42:42.660009+00:00 [running]> on host 6be8c7a30c6f
[2021-02-06 06:42:45,643] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=runme_2
AIRFLOW_CTX_EXECUTION_DATE=2021-02-06T06:42:42.660009+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-02-06T06:42:42.660009+00:00
[2021-02-06 06:42:45,644] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-02-06 06:42:45,645] {bash.py:158} INFO - Running command: echo "example_bash_operator__runme_2__20210206" && sleep 1
[2021-02-06 06:42:45,654] {bash.py:169} INFO - Output:
[2021-02-06 06:42:45,655] {bash.py:173} INFO - example_bash_operator__runme_2__20210206
[2021-02-06 06:42:46,656] {bash.py:177} INFO - Command exited with return code 0
[2021-02-06 06:42:46,684] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_2, execution_date=20210206T064242, start_date=20210206T064245, end_date=20210206T064246
[2021-02-06 06:42:46,720] {taskinstance.py:1219} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-02-06 06:42:46,745] {local_task_job.py:146} INFO - Task exited with return code 0
